tedana/stats.py

```diff
@@ -74,28 +135,20 @@ def computefeats2(data, mmix, mask=None, normalize=True):
     # demean masked data
     if mask is not None:
         data = data[mask, ...]
+    # normalize data (minus mean and divide by std)
     data_vn = stats.zscore(data, axis=-1)
 
-    # get betas of `data`~`mmix` and limit to range [-0.999, 0.999]
-    data_R = get_coeffs(data_vn, mmix, mask=None)
-    data_R[data_R < -0.999] = -0.999
-    data_R[data_R > 0.999] = 0.999
-
-    # R-to-Z transform
-    data_Z = np.arctanh(data_R)
+    # get betas and z-values of `data`~`mmix`
+    # mmix is normalized internally
```

Comment:
I think we should do it here anyway.

Reply:
What if we use `add_const=True` while calling `get_ls_coeffs` instead, now that there's the option? I think it would make more sense given how the program runs.

In fact, I would set add_const default as True, given that if the data is not demeaned, you need the intercept, and if it's demeaned, the intercept doesn't bother you (it's 0).

@CesarCaballeroGaudes is there any other difference between normalising before computing LS and adding the intercept in the model?

Reply:
I like including the intercept by default, but isn't variance normalization required to get standardized parameter estimates?